root_main.tf

```diff
@@ -44,63 +53,75 @@ module "encryption_key" {
 }
 
 module "log_data_sns" {
-  source         = "./tdr-terraform-modules/sns"
-  apply_resource = local.environment == "mgmt" || local.environment == "intg" || local.environment == "staging" || local.environment == "prod" || local.environment == "sbox" ? true : false
-  project        = var.project
-  common_tags    = local.common_tags
-  function       = "logs"
-  sns_policy     = "log_data"
+  source = "git::https://github.com/nationalarchives/da-terraform-modules//sns"
```

Comment:
Would da-terraform-modules want to be added a sub-module so don't have to call its as a https?